src/test/unit/function-info.test.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { stripFunctionArguments } from '../../profile-logic/function-info';
+
+describe('get-function-name', function () {
+  it('should strip the function arguments', function () {
+    expect(stripFunctionArguments('ns::fn()')).toEqual('ns::fn');
+    expect(stripFunctionArguments('ns::fn(bool (*)(JS::Handle<JSObject*>))')).toEqual('ns::fn');
+    expect(stripFunctionArguments('Bar* ns::fn<(Foo)0>(void const*, unsigned int)')).toEqual('Bar* ns::fn<(Foo)0>');
```

Comment:
> I think "copy function name" should copy js::detail::HashTable::lookupForAdd and get rid of the template information.
> 
> What do you think @mstange ?

Hard to say. I agree that in this particular case, `js::detail::HashTable::lookupForAdd` is *much* easier to read. but in the general case it might be getting rid of crucial information. So I'd say let's not touch template parameters for now.